adoc/quickstart3_chap_install_overview.adoc

```diff
@@ -127,7 +127,7 @@ Both methods of installation will be covered in this guide.
 [[quickstart.sect.software.requirements]]
 == Hardware Requirements
 
-Review the following table for {productname} hardware and software requirements.
+Review the following table for {productname} hardware and software requirements on {x86_64} and IBM {ppc} architecture.
```

Comment:
Vendor and architecture should be one entity imho and use {ppc64le} just to make it consistent with {x86_64}.